apps/src/netsim/NetSimRemoteNodeSelectionPanel.html.ejs

```diff
@@ -195,7 +195,7 @@ function writeNodeRow(nodeID, nodeName, nodeStatus, buttonType, addlClass) {
           writeHeader(i18n.outgoingConnectionRequests());
           outgoingRequestRows.forEach(function (row) {
             var outgoingStatus = i18n.lobbyStatusWaitingForOther({
-              spinner: '<img src="/assets/spinner-big.gif" />',
+              spinner: '<img src="' + assetUrl('media/turtle/loading.gif') + '" />',
```

Comment:
it seems odd to be reaching around to the turtle assets.. should this image be in a more common location?